React/prop-types.md

```diff
@@ -0,0 +1,82 @@
+# Prop Types
+
+## Learning Goals
+By the end of this lesson you should be able to:
+
+- Explain the use of `PropTypes`
+- Use `PropTypes` to define and document the props a component can take
+
+You may have noticed this message from your JavaScript linter.  "____ is missing from props validation (react/prop-types)."  The linter is encouraging you to use a feature of React called Prop-types in which you list the props a component takes and their types.
```

Comment:
Can we make the "Prop-types" either all lowercase or camel case?

Reply:
Updated